### PR TITLE
Update de.m3u

### LINKED ISSUE #12613: Add: Folx Music Television [DE]
### Channel ID

FolxMusicTelevision.de

### Stream URL (optional)

https://cdne.folxplay.tv/folx-trz/streams/ch-4/master.m3u8

### Notes (optional)

This link is working.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #12614: Add: Zwei Music Television [DE]
### Channel ID

ZweiMusicTelevision.de

### Stream URL (optional)

https://cdne.folxplay.tv/folx-trz/streams/ch-2/master.m3u8

### Notes (optional)

This stream is working.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #12615: Add: One Music Television [DE]
### Channel ID

OneMusicTelevision.de

### Stream URL (optional)

https://cdne.folxplay.tv/folx-trz/streams/ch-3/master.m3u8

### Notes (optional)

This stream is works.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #12616: Add: Adria Music Television [DE]
### Channel ID

AdriaMusicTelevision.de

### Stream URL (optional)

https://cdne.folxplay.tv/folx-trz/streams/ch-1/master.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #12617: Add: One Adria [DE]
### Channel ID

OneAdria.de

### Stream URL (optional)

https://cdne.folxplay.tv/folx-trz/streams/ch-6/master.m3u8

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by Carlinhos027**
Deleted duplicate stream, corrected IDs, replaced links & added new channels

Closes #12613 
Closes #12614 
Closes #12615 
Closes #12616 
Closes #12617

### CODE REVIEW

File: streams/de.m3u [hunk 1]
```diff
@@ -140,11 +138,11 @@ https://ub1doy938d.gjirafa.net/live/oEZi1otQqIHglY0tzZidyCPWhxrS5WgO/yky0y1.m3u8
 http://51.15.2.151/hls/kanalavrupa.m3u8
 #EXTINF:-1 tvg-id="KaufBeiTV.de",KaufBei TV (576p)
 https://api.alpaca.t62a.com/hls/9103/index.m3u8
-#EXTINF:-1 tvg-id="KIKA.de",KiKA [Geo-blocked]
+#EXTINF:-1 tvg-id="KIKA.de",KiKA (1080p) [Geo-blocked]
 https://kikageohls.akamaized.net/hls/live/2022693/livetvkika_de/master.m3u8
```
Comment (BellezaEmporium) [inline streams/de.m3u:142]: There's also a worldwide version of Kika, heavily censored for copyright reasons but exists : https://kikahls.akamaized.net/hls/live/2022690/livetvkika_ww/master.m3u8

File: streams/de.m3u [hunk 2]
```diff
@@ -470,21 +468,47 @@ https://edge14.vedge.infomaniak.com/livecast/ik:wir24tv_hms/manifest.m3u8
...
+#EXTINF:-1 tvg-id="DeluxeDance.de",Deluxe Dance (1080p) [Geo-blocked]
+https://sdn-global-live-streaming-packager-cache.3qsdn.com/64733/64733_264_live.m3u8
+#EXTINF:-1 tvg-id="AstroTV.de",AstroTV (1080p) [Not 24/7]
+https://ythls.onrender.com/channel/UCwS3E2NaN3uorr9wf2_i6Cg.m3u8
+#EXTINF:-1 tvg-id="PearlTV.de",Pearl TV (2160p) [Not 24/7]
+https://ythls.onrender.com/channel/UCp0urejlDaYz4G8xpTxdIvw.m3u8
+#EXTINF:-1 tvg-id="AristoTV.de",Aristo TV (360p)
+https://ma.anixa.tv/clips/stream/aristo/index.m3u8
+#EXTINF:-1 tvg-id="AnixePlus.de",Anixe + (360p)
+https://ma.anixa.tv/clips/stream/anixesd/index.m3u8
```
Comment (BellezaEmporium) [inline streams/de.m3u:504]: https://ma.anixa.tv/clips/stream/anixehd/index.m3u8 is Eazzzy TV, if you want to add it.

Comment (Carlinhos027) [reply streams/de.m3u:504]: Is already added.

File: streams/de.m3u (hunk 1 above)
Comment (Carlinhos027) [reply streams/de.m3u:142]: Added. Thx bro

**Review by UltraHDR — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**